src/applications/personalization/dashboard/tests/components/Dashboard.unit.spec.jsx

```diff
@@ -330,4 +330,54 @@ describe('<Dashboard />', () => {
       expect(getByTestId('req-loader')).to.exist;
     });
   });
+
+  it('renders a link to confirm contact email', async () => {
+    mockFetch();
+    const { getByTestId } = renderInReduxProvider(<Dashboard />, {
+      initialState,
+      reducers,
+    });
+    const testId = 'va-profile--confirm-contact-email-link';
+    const href = '/profile/contact-information#contact-email-address';
+    const textContent = /Confirm your contact email address/;
+    await waitFor(() => {
+      const confirmEmailLink = getByTestId(testId);
+      expect(confirmEmailLink).to.exist;
+      expect(confirmEmailLink.link).to.equal(href);
+      expect(confirmEmailLink.text).to.match(textContent);
+    });
+  });
+
+  it('suppresses the email confirmation link when CONTACT_EMAIL_CONFIRMED cookie is set', () => {
```

Comment:
I think this test should do the date check rather than cookie check, now that we've moved away from the cookie.